src/Symfony/Component/HttpKernel/Log/DebugLoggerConfigurator.php

```diff
@@ -30,7 +30,16 @@ public function __construct(callable $processor, ?bool $enable = null)
     public function pushDebugLogger(Logger $logger): void
     {
         if ($this->processor) {
+            $processors = $logger->getProcessors();
+            while ([] !== $logger->getProcessors()) {
+                $logger->popProcessor();
+            }
+
+            // Ensure the DebugLogger is the first processor as Monolog add processors in reverse order
             $logger->pushProcessor($this->processor);
+            foreach ($processors as $processor) {
+                $logger->pushProcessor($processor);
+            }
```

Comment:
For me this would deserve its own PR.. as this one only says "add extra data" and not "changes the order of procesors" :)

Reply:
Indeed, that's why I'm not 100% sure about this change. But if processors are left as is, `DebugProcessor` is the very last to be pushed to the list, and it will not get extra data added by next processors (because of reverse order used by Monolog).

Do you see another way to do so ? Working on `CompilerPass` looks pretty complex.